crm_claim_type/models/crm_claim.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2015 Vauxoo: Yanina Aular <yani@vauxoo.com>,
+#                        Osval Reyes <osval@vauxoo.com>
+# Copyright 2017 Bhavesh Odedra <bodedra@ursainfosystems.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class CrmClaim(models.Model):
+
+    """
+    CRM Claim
+    """
+
```

Comment:
This can be removed as it's not adding anything useful:

```suggestion
```

Reply:
Fixed.